src/app/technical-record/tyres/tyres.component.spec.ts

```diff
@@ -49,25 +50,18 @@ describe('VehicleSummaryComponent', () => {
     fixture = TestBed.createComponent(VehicleSummaryComponent);
     injector = getTestBed();
     component = fixture.componentInstance;
-    component.activeRecord = {
-      'vin': 'XMGDE02FS0H012345',
-      'vehicleSize': 'small',
-      'testStationName': 'Rowe, Wunsch and Wisoky',
-      'vehicleId': 'JY58FPP',
-      'vehicleType': 'psv',
-      'axles': [
-        { 'parkingBrakeMrk': false, 'axleNumber': 1 },
-        { 'parkingBrakeMrk': true, 'axleNumber': 2 },
-        { 'parkingBrakeMrk': false, 'axleNumber': 3 }
-      ]
-    };
+    component.activeRecord = TESTING_UTILS.mockTechRecord();
```

Comment:
pass in tyre fields so that we it is picked up in snapshots

Reply:
axle has tyre fields. So passed in the axle with the tyre fields.